src/server/kernel/woutput.h

```diff
@@ -45,6 +46,9 @@ class WAYLIB_SERVER_EXPORT WOutput : public QObject, public WObject
     Q_PROPERTY(Transform orientation READ orientation NOTIFY orientationChanged)
     Q_PROPERTY(float scale READ scale NOTIFY scaleChanged)
     Q_PROPERTY(bool forceSoftwareCursor READ forceSoftwareCursor WRITE setForceSoftwareCursor NOTIFY forceSoftwareCursorChanged)
+    Q_PROPERTY(int x READ x WRITE setX NOTIFY xChanged)
```

Comment:
Why? The output's position is control in QWOutputLayout, Don't set it in WOutput, you can control position by `OutputItem`.